packages/dashboard/src/components/tasks/tasks-app.tsx

```diff
@@ -191,8 +191,8 @@ export const TasksApp = React.memo(
             filterColumn && filterColumn === 'id_' ? filterValue : undefined,
             filterColumn && filterColumn === 'category' ? filterValue : undefined,
             filterColumn && filterColumn === 'requester' ? filterValue : undefined,
-            filterColumn && filterColumn === 'label=pickup' ? filterValue : undefined,
-            filterColumn && filterColumn === 'label=destination' ? filterValue : undefined,
+            // filterColumn && filterColumn === 'label=pickup' ? filterValue : undefined,
+            // filterColumn && filterColumn === 'label=destination' ? filterValue : undefined,
```

Comment:
remove them instead of commenting out?

Reply:
ah thanks for catching that, I was in the midst of making sure everything built and haven't baked in the labels logic yet, will do so

Reply:
https://github.com/open-rmf/rmf-web/pull/971/commits/a2c44e8182287e1d55e754faba20e9be08023597